include/ibm/management_console_rest.hpp

```diff
@@ -926,7 +926,8 @@ inline void getCSREntryAck(const std::shared_ptr<bmcweb::AsyncResp>& asyncResp,
                              const std::variant<std::string>& hostAck) {
         if (ec)
         {
-            BMCWEB_LOG_ERROR << "getCSREntryAck respHandler got error " << ec;
+            BMCWEB_LOG_ERROR << "getCSREntryAck respHandler got"
+                             << "error: " << ec;
```

Comment:
I think it is better to be a single "string" rather than separating it into 2 strings.

```
BMCWEB_LOG_ERROR << "getCSREntryAck respHandler got error " << ec;
```

`clang-format-16` will appropriately format the statement if necessary.

Reply:
clang is not making these changes somehow , but every where else this format is followed thats why i made the changes

Reply:
Then, please keep the original as what clang-format-16 does. 
There seems no strong reason to break one string into 2.